src/main/java/appeng/services/version/ModVersionFetcher.java

```diff
@@ -30,11 +53,18 @@ public Version get()
 		{
 			return new DoNotCheckVersion();
 		}
-		else
+
+		try
 		{
 			final Version version = this.parser.parse( this.rawModVersion );
 
 			return version;
 		}
+		catch( final Exception e )
```

Comment:
Considering something similar already. Basically replacing all assertion through checked exceptions, but I wanted to hear your feedback first as you implemented the version checker.